stdlib/Profile/test/runtests.jl

```diff
@@ -204,6 +204,24 @@ end
     @test getline(values(fdictc)) == getline(values(fdict0)) + 2
 end
 
+import InteractiveUtils
+
+@testset "Module short names" begin
+    Profile.clear()
+    @profile InteractiveUtils.peakflops()
+    io = IOBuffer()
+    ioc = IOContext(io, :displaysize=>(1000,1000))
+    Profile.print(ioc, C=true)
+    str = String(take!(io))
+    slash = Sys.iswindows() ? "\\" : "/"
+    @test occursin("@Compiler" * slash, str)
```

Comment:
These tests have been seen failing quite frequently in CI since merging. Is there something wrong with the PR or test design?

Reply:
Somehow the profiler sometimes misses all of the compilation. Perhaps `peakflops` could be replaced with something like putting a sleep in a generated function eval. Or just test the underlying functions more precisely.